components/cdp_traits/lib.rs

```diff
@@ -0,0 +1,24 @@
+extern crate futures;
+extern crate script_traits;
+
+use futures::sync::mpsc as futures_mpsc;
+use script_traits::ConstellationMsg;
+use std::sync::mpsc as std_mpsc;
+
+/// Establish a CDP server control channel.
+#[inline]
```

Comment:
Why the inline marking? I can't imagine that this is a performance hit if it's not there.